CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Braintree iOS SDK Release Notes
 
+## unreleased
+* BraintreePayPalNativeCheckout (BETA)
```

Comment:
When are we planning to remove the (BETA) tag? On the next NXO release?

Reply:
The plan is to remove the beta tag once we have the GA version in the SDK! It sounds like there may be a fix version before the GA but we can ask today in our meeting with them.